ci/rebuild-docker-images.py

```diff
@@ -485,6 +488,98 @@ def playwright_build(distro, arch):
 
 
 
+def appium_build(distro, arch):
+    tag = f"{registry_base}{distro[0]}-{distro[1]}"
+    build_tag(tag, arch, f"""
+FROM openjdk:18-ea-11-jdk-slim-bullseye
+
+ENV HOME=/root \
+    LANG=en_US.UTF-8 \
+    DEBIAN_FRONTEN=noninteractive \
+    LANGUAGE=en_US.UTF-8 \
+    LC_ALL=C.UTF-8 \
+    DISPLAY=:0.0 \
+    DISPLAY_WIDTH=1920 \
+    DISPLAY_HEIGHT=900
+
+
+WORKDIR /
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update
+RUN apt install -y ca-certificates curl git cpu-checker supervisor vim bash wget unzip xvfb x11vnc fluxbox xterm novnc net-tools htop libpulse-dev libnss3 libxcursor1 libasound2 libqt5gui5 libc++-dev libxcb-cursor0 htop tree tar gzip
```

Comment:
```suggestion
RUN apt-get -o=Dpkg::Use-Pty=0 -q install -y ca-certificates curl git cpu-checker supervisor vim bash wget unzip xvfb x11vnc fluxbox xterm novnc net-tools htop libpulse-dev libnss3 libxcursor1 libasound2 libqt5gui5 libc++-dev libxcb-cursor0 htop tree tar gzip
```

- `apt-get` for non-interactive commands (`apt` is meant for humans, `apt-get` for scripts (better log output, less human-pretty formatting); also `apt` warns when used non-interactively).
- `-o=Dpkg::Use-Pty=0` so that interactive package configs can't stop and ask questions